### PR TITLE
Updated Hemato DB provider (#118)

### PR DESCRIPTION
**Pull request by bekeles**

### CODE REVIEW

**Comment by ctmay4** [comment]
This looks like it is still missing the public constructor for `MphUtils` so we can support multiple instances with different providers.

File: src/main/java/com/imsweb/mph/MphUtils.java [hunk 1]
```diff
@@ -279,6 +279,18 @@ else if (result.getFinalResult() != null) {
...
 
+    /**
+     * Calculates if two hematopoietic diseases are same primaries based on Hemato DB.
+     * @param morph1 disease 1
+     * @param morph2 disease 2
+     * @param year1 diagnosis year 1
+     * @param year2 disgnosis year 2
+     * @return true if two diseases are same primary and false otherwise.
+     */
+    public boolean isHematoSamePrimary(String morph1, String morph2, int year1, int year2) {
```
Comment (depryf) [inline src/main/java/com/imsweb/mph/MphUtils.java:290]: There are two things requested in the issue that weren't done:

1. Making the constructor public.
2. Fixing the warnings in this class (I saw a bunch of them when I opened it in Intellij).

**Review by depryf — CHANGES_REQUESTED** [review]

File: src/main/java/com/imsweb/mph/MphUtils.java (hunk 1 above)
Comment (bekeles) [reply src/main/java/com/imsweb/mph/MphUtils.java:290]: This is fixed.

**Review by depryf — APPROVED** [review]

**Comment by depryf** [comment]
@bekeles there is a merge conflict (probably the changelog file). Can you please merge the main branch and resolve the conflict? Thanks.

**Comment by bekeles** [comment]
Merge conflict has been resolved.

**Pull request merged**